source/options.css

```diff
@@ -2,12 +2,28 @@ body {
 	min-width: 400px; /* Required to size the Browser Action popup */
 }
 textarea {
+	display: block;
 	width: 100%;
+	font-family: monospace !important;
+	font-size: 12px;
+	line-height: 1.5;
+	resize: vertical;
+}
+label {
+	display: flex;
+	align-items: center;
+}
```

Comment:
⬆️❓

Reply:
| Before | After |
| ------- | ----- |
| ![image](https://user-images.githubusercontent.com/12368291/37667487-2793437c-2c88-11e8-9913-f5e06d9ad97d.png) | ![image](https://user-images.githubusercontent.com/12368291/37667537-3c8456ae-2c88-11e8-810d-b8442721a991.png) |

Just some vertical alignment!

Reply:
`vertical-alignment: ---px` on the checkbox